pkg/plugins/golang/v4/scaffolds/internal/templates/main.go

```diff
@@ -33,6 +33,8 @@ type Main struct {
 	machinery.BoilerplateMixin
 	machinery.DomainMixin
 	machinery.RepositoryMixin
+
+	ControllerRuntimeVersion string
```

Comment:
Just to allow we automatically bump the link for the docs with the version used to do the scaffold.
As we do in other places